packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -577,11 +578,15 @@ const Services = WebexPlugin.extend({
     }
 
     if (catalog.isReady) {
-      return url ?
-        Promise.resolve(url) :
-        Promise.reject(new Error(
-          `services: service '${name}' was not found in any of the catalogs`
-        ));
+      if (url) return Promise.resolve(url);
```

Comment:
Not sure if the ESLint config enforces this, but shouldn't this if statement be wrapped in `{` and `}`? JS does support this syntax, but wanting confirmation that ESLint does as well.

Reply:
done